workflows/ensemblgenedownload.nf

```diff
@@ -49,13 +44,34 @@ workflow ENSEMBLGENEDOWNLOAD {
 
     ch_versions = Channel.empty()
 
-    //
-    // SUBWORKFLOW: Read in samplesheet, validate and stage input files
-    //
-    INPUT_CHECK (
-        ch_input
+    PARAMS_CHECK (
+        [
+            params.input,
+            params.assembly_accession,
+            params.ensembl_species_name,
+            params.geneset_version,
+            params.outdir,
+        ]
+    )
```

Comment:
Pass input as a single channel instead of the `params` directly.

Reply:
I don't understand, this is already a single channel cf https://github.com/sanger-tol/ensemblgenedownload/pull/1/files/62f6372fea1a31272240649a6176735f03f6453b#diff-9165b3ed0bbaa6fe730b0ca414c560296450089f61184965c4c6cd3da2ef5bf6R10

Reply:
OK, I think I understand what you mean. I'm getting close to a version where the four parameters are separated from the samplesheet parameter. They're bundled in a tuple that is passed as another channel. I'll push it this morning